lib/wikidata/fetcher.rb

```diff
@@ -7,10 +7,16 @@
 
 class WikiData
   class Fetcher < WikiData
+    LOOKUP_FILE = 'https://raw.githubusercontent.com/everypolitician/wikidata-fetcher/master/lookup.json'.freeze
+
     def self.find(ids)
       Hash[Wikisnakker::Item.find(ids).map { |item| [item.id, new(item: item)] }]
     end
 
+    def self.lookup
+      @lookup ||= JSON.parse(open(LOOKUP_FILE).read, symbolize_names: true)
```

Comment:
I wonder if there's a more suitable name for this? The more natural reading of a method called "lookup" is that it's a verb, and needs an argument, rather than being a noun to represent the data structure it's returning.